spark/src/main/java/com/netflix/iceberg/spark/source/Writer.java

```diff
@@ -242,6 +244,15 @@ private Path lazyDataPath() {
                   .named("table")
                   .build();
 
+            case ORC: {
+              @SuppressWarnings("unchecked")
+              SparkOrcWriter writer = new SparkOrcWriter(ORC.write(file)
+                  .schema(schema)
+                  .partitionSpec(spec)
```

Comment:
I think spec can be removed because it isn't relevant at a the level of a single file. It also isn't used in the `OrcFileAppender`.

Reply:
ok